resources/views/Checkout/OrderDetails.twig

```diff
@@ -160,17 +163,43 @@
                                 </div>
                             {% endif %}
 
+
+
+                            {% if item.amounts[0].discount > 0 %}
+                                {% if item.amounts[0].isPercentage ==  "1" %}
+                                    {% set rebatedAmountItemGross = ((item.amounts[0].priceGross  / (100 - item.amounts[0].discount)) * 100) - item.amounts[0].priceGross %}
```

Comment:
Kann das setzen und berechnen in einen entsprechenden Context ausgelagert werden?